ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
I would like to have some place for ci-independent build scripts that will not get contaminated with our ci-specific stuff to the point that it would be impossible to use them outside of our ci.